src/circuit/gadget/ecc/chip/mul/complete.rs

```diff
@@ -70,8 +70,8 @@ impl Config {
                 // k_i = 0 => y_p = -base_y
                 // k_i = 1 => y_p = base_y
                 let y_switch = {
-                    let k_minus_one = k.clone() - Expression::Constant(pallas::Base::one());
-                    k_minus_one * (base_y.clone() + y_p.clone()) + k * (base_y - y_p)
+                    let one_minus_k = Expression::Constant(pallas::Base::one()) - k.clone();
+                    one_minus_k * (base_y.clone() + y_p.clone()) + k * (base_y - y_p)
```

Comment:
It occurs to me that we should probably also be consistent about expression ordering for these "either-or" constraints: either `(1 - b) * false_case + b * true_case` or `b * expr_if_true + (1 - b) * expr_if_false` (depending on whether we want it to read like a case expression, or a ternary expression).

Non-blocking; this cleanup should go into a separate PR (as it would also be nice to consider how we could ensure that `b` is a boolean-constrained expression).

Reply:
This is not a correct change.

Reply:
Sidenote: I don't think the complete addition constraints for variable-base scalar mul are currently documented in our book. https://zcash.github.io/orchard/design/circuit/gadgets/ecc/var-base-scalar-mul.html#circuit-design

Reply:
Note that this change is correct because `y_switch` is being constrained to 0 (when `q_mul_decompose_var` is selected), and so sign does not matter.